frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/EditConfigurationModal/helpers.tests.ts

```
@@ -0,0 +1,253 @@
+  });
+
+  it("returns error when root element is not <dict>", () => {
+    const error = validateXml("<array><string>hi</string></array>");
+    expect(error).toMatch(/root element must be <dict>/i);
+  });
+
+  it("returns null for plist-wrapped dict", () => {
+    expect(
+      validateXml(
+        "<plist><dict><key>k</key><string>v</string></dict></plist>"
+      )
```

ESLint

[error] Replace `⏎········"<plist><dict><key>k</key><string>v</string></dict></plist>"⏎······` with `"<plist><dict><key>k</key><string>v</string></dict></plist>"`

(prettier/prettier)